pkgs/applications/graphics/icon-library/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchurl, wrapGAppsHook
+, cargo, desktop-file-utils, meson, ninja, pkg-config, python3, rustc
+, dbus, gdk-pixbuf, glib, gtk3, gtksourceview4, libhandy
+}:
+
+stdenv.mkDerivation rec {
+  pname = "icon-library";
+  version = "0.0.8";
+
+  src = fetchurl {
+    url = "https://gitlab.gnome.org/World/design/icon-library/uploads/fdf890706e0eef2458a5285e3bf65dd5/icon-library-${version}.tar.xz";
```

Comment:
Could we use https://gitlab.gnome.org/World/design/icon-library/-/archive/0.0.8/icon-library-0.0.8.tar.gz instead or are the archives different? Then updates would be automatic.

Reply:
The generated archives are what `fetchFromGitLab` uses and those do not contain vendored dependencies so we would need `cargoSetupHook`. Maybe that is better, though.

Reply:
I would rather use a static tarball than a dynamic one where possible because of the reproducibility advantages, and especially in this case because otherwise we'd have to re-vendor the dependencies ourselves.